### PR TITLE
ENH: Add python wrapping for CudaCommon

### PR DESCRIPTION
**Pull request by AlexyPellegrini**
This wrapping differs from the one that used to be in RTK by the fact that it only wraps the interface for floating type data and only for image dimensions included in `${ITK_WRAP_IMAGE_DIMS}` and vector components included in `${ITK_WRAP_VECTOR_COMPONENTS}`.
RTK will have to wrap by itself everything else it needs.
This is related to CudaCommon migration out of RTK.

### CODE REVIEW

**Comment by AlexyPellegrini** [comment]
@LucasGandel

File: wrapping/itkCudaInPlaceImageFilter.wrap [hunk 1]
```diff
@@ -1,40 +1,31 @@
-if(RTK_USE_CUDA)
+itk_wrap_class("itk::CudaInPlaceImageFilter" POINTER)
```
Comment (SimonRit) [inline wrapping/itkCudaInPlaceImageFilter.wrap:1]: Same comment as above

**Comment by SimonRit** [comment]
I think the best strategy would be to mimick the wrapping of GPUCommon, there is no reason to limit the wrapping to float. See [here](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Modules/Core/GPUCommon/wrapping).

File: wrapping/itkCudaImageToImageFilter.wrap [hunk 2]
```diff
@@ -1,22 +1,15 @@
-if(RTK_USE_CUDA)
+itk_wrap_class("itk::CudaImageToImageFilter" POINTER)
 
-  itk_wrap_class("itk::CudaImageToImageFilter" POINTER)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    itk_wrap_template("CI${ITKM_F}${d}CI${ITKM_F}${d}"
+      "itk::CudaImage<${ITKT_F}, ${d}>, itk::CudaImage<${ITKT_F}, ${d}>")
```
[the remark this answers is not in the document]
Comment (LucasGandel) [reply wrapping/itkCudaImageToImageFilter.wrap:5]: Agreed, it can be removed. It does not make sense to use the default filter.

**Comment by LucasGandel** [comment]
> I think the best strategy would be to mimick the wrapping of GPUCommon, there is no reason to limit the wrapping to float. See [here](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Modules/Core/GPUCommon/wrapping).

Sounds good, we will add the following types (from itkGPUImage.wrap) :
```
UNIQUE(types "UC;UL;${ITKM_IT};${WRAP_ITK_SCALAR}")
UNIQUE(vector_types "${WRAP_ITK_VECTOR_REAL};${WRAP_ITK_COV_VECTOR_REAL}")
```

**Comment by AlexyPellegrini** [comment]
I applied the modifications related to the previous comments. Let me know if you think something should be changed again :)

**Review by SimonRit — APPROVED** [review]
The structure slighly differs from GPUCommon but that looks good to me. Thanks!

**Review by LucasGandel — APPROVED** [review]

**Review by LucasGandel — APPROVED** [review]

**Review by LucasGandel — APPROVED** [review]

**Pull request merged**